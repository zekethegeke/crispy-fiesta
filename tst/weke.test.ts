import { double } from 'weke-plex';

test('adds 2 * 2  to equal 4', () => {
	expect(double(2)).toBe(4);
});
	
test('adds 2 * 2  to equal 4', () => {
	expect(double(2)).toMatchSnapshot();
});
