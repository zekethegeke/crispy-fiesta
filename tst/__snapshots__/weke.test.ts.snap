// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds 2 * 2  to equal 4 1`] = `4`;
